saleor/graphql/checkout/mutations/utils.py

```diff
@@ -303,3 +381,36 @@ def check_permissions_for_custom_prices(app, lines):
         not app or not app.has_perm(CheckoutPermissions.HANDLE_CHECKOUTS)
     ):
         raise PermissionDenied(permissions=[CheckoutPermissions.HANDLE_CHECKOUTS])
+
+
+def find_line_id_when_variant_parameter_used(variant_id, lines_info):
+    """Return line id by using provided variantId parameter.
```

Comment:
IMHO, would be good if you  could add a note here, that you're expecting `DB ID` for variant ID (It is `graphql/` directory so this could be tricky for someone).